src/amo/components/CollectionManager/index.js

```diff
@@ -317,11 +345,21 @@ export class CollectionManagerBase extends React.Component<Props, State> {
             value={this.state.slug}
           />
         </div>
-        {this.state.addonAddedStatus === ADDON_ADDED_STATUS_SUCCESS && (
-          <Notice type="success">
-            {i18n.gettext('Added to collection')}
-          </Notice>
-        )}
+
+        <ReactCSSTransitionGroup
```

Comment:
I forgot how to use this component.. do we have to keep it all the time in the UI tree? (or can we put it in the condition where `Notice` is).

Reply:
yes - it has to always be in the DOM to work